internal/controller/ols_app_server_deployment.go

```diff
@@ -386,3 +438,18 @@ func (r *OLSConfigReconciler) dataCollectorEnabled(cr *olsv1alpha1.OLSConfig) (b
 	}
 	return configEnabled && telemetryEnabled, nil
 }
+
+// extraCertsDefined checks if the ConfigMap named ols-extra-ca exists and returns it if so
+func (r *OLSConfigReconciler) extraCertsDefined(namespace string) (bool, *corev1.ConfigMap, error) {
+	configMap := &corev1.ConfigMap{}
+	err := r.Client.Get(context.TODO(), client.ObjectKey{
+		Name:      "ols-extra-ca",
```

Comment:
I suggest we leave the configmap name configurable in the OLSConfig CR. It is difficult for users to memorize these constants.